x/did/keeper/did.go

```diff
@@ -7,15 +7,15 @@ import (
 )
 
 func (k Keeper) SetDIDDocument(ctx sdk.Context, did string, doc types.DIDDocumentWithSeq) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.DIDKey))
-	key := DIDDocumentKey(did)
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.DIDKeyPrefix)
+	key := []byte(did)
```

Comment:
The `keeper/keeper_keys.go` in the `x/did` is not necessary anymore. From Stargate, the definition of keeper key prefixes has been moved to the `types/keys.go` (`DIDKeyPrefix`). 

In the current implementation in the `stargate` branch, the keeper key is double-prefixed (by [DID-value-](https://github.com/medibloc/panacea-core/blob/f3d98c87dee37f4fb43b85f6b16116ced55fed1e/x/did/types/keys.go#L25) and [0x11](https://github.com/medibloc/panacea-core/blob/f3d98c87dee37f4fb43b85f6b16116ced55fed1e/x/did/keeper/keeper_keys.go#L10)). We've missed it.

Reply:
Ah. We where using a duplicate prefix! 
I understand.